braininventory/get.py

```diff
@@ -100,3 +100,5 @@ def report():
 
 	return report
 
+df['generalmodality'].value_counts()
```

Comment:
This is correct but I want you to do is create a function from this line. Something that looks like

```
def get_number_of_modalities(df):
    return df['generalmodality'].value_counts()
```
Though what I suggested above is accurate I would like you to

* convert your statement into a [dictionary](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_dict.html)
* Document the function.